ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/NodePage.tsx

```diff
@@ -101,43 +95,19 @@ function NodePage() {
                             component={TabsComponent.nav}
                             className="pf-v5-u-pl-md pf-v5-u-background-color-100"
                             role="region"
-                            mountOnEnter
-                            unmountOnExit
                         >
-                            <Tab
-                                eventKey={vulnTabKey}
-                                title={<TabTitleText>Vulnerabilities</TabTitleText>}
-                                tabContentId={vulnTabId}
-                                tabContentRef={vulnTabRef}
-                            />
-                            <Tab
-                                eventKey={detailsTabKey}
-                                title={<TabTitleText>Details</TabTitleText>}
-                                tabContentId={detailsTabId}
-                                tabContentRef={detailsTabRef}
-                            />
+                            <Tab eventKey={vulnTabKey} title={vulnTabKey} />
+                            <Tab eventKey={detailTabKey} title={detailTabKey} />
                         </Tabs>
                     </PageSection>
-                    {activeTabKey === vulnTabKey && (
-                        <TabContent
-                            id={vulnTabId}
-                            ref={vulnTabRef}
-                            eventKey={vulnTabKey}
-                            className="pf-v5-u-display-flex pf-v5-u-flex-direction-column pf-v5-u-flex-grow-1"
-                        >
-                            <NodePageVulnerabilities />
-                        </TabContent>
-                    )}
-                    {activeTabKey === detailsTabKey && (
-                        <TabContent
-                            id={detailsTabId}
-                            ref={detailsTabRef}
-                            eventKey={detailsTabKey}
-                            className="pf-v5-u-display-flex pf-v5-u-flex-direction-column pf-v5-u-flex-grow-1"
-                        >
-                            <NodePageDetails />
-                        </TabContent>
-                    )}
+                    <PageSection
+                        isFilled
+                        padding={{ default: 'noPadding' }}
+                        className="pf-v5-u-display-flex pf-v5-u-flex-direction-column"
+                    >
+                        {activeTabKey === vulnTabKey && <NodePageVulnerabilities />}
+                        {activeTabKey === detailTabKey && <NodePageDetails />}
```

Comment:
The main functionality of the TabContent wrapper went against the grain in a couple of ways:
- It unconditionally rendered its content, regardless of the selected tab, which would cause requests in child components to fire early
- It shows/hides the tab based on the presence of the `hidden` attribute, but since we needed to apply `display: flex` to the component here for layout, the tab content sections were always all displayed regardless of selected tab.

Due to both issues above we need to conditionally render the content based on the selected key, and since the majority of the functionality we got from `TabContent` was unused, removing the component cut down on a lot of verbosity.

That said, there are some aria attributes that we lose out of the box by not using the wrapper so I'd be in favor of adding it back in.

Reply:
Sounds like it may be better to report our problems with it to the PatternFly team, and go with what you got.

Maybe add as much aria-* as you can in a follow-up, along with a comment with a shortened explanation.

Reply:
Sounds good. I think our issues are specific to the TabContent not being a child of the `<Tabs>` component. They have props that do what we need, but they don't work in this specific case. I'm not sure if that is an intentional design decision, a limitation, etc. I made a note to look into it more and follow up with the PF team.